api/src/main/java/io/opentelemetry/metrics/LongMeasure.java

```diff
@@ -69,7 +69,7 @@
    * @since 0.1.0
    */
   @ThreadSafe
-  interface BoundLongMeasure extends BoundInstrument {
+  interface BoundLongMeasure extends InstrumentWithBinding.BoundInstrument {
```

Comment:
Please send these changes in a different PR, they are javadoc warnings, saw them in the CI.

Reply:
Sure thing. Need to get the build working on CI, too.

Reply:
Don't think for that small PR you need to fix anything :)

Reply:
Without that fix, this change doesn't work, for reasons I don't understand.

Reply:
But I can do it first thing tomorrow.

Reply:
done!